timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -39,7 +39,6 @@ class SimpleAbstractTrainer:
 
```

Comment:
@Innixma Around the v0.5.0 release, we planned to move the `SimpleAbstractTrainer` class to `autogluon.core`, which would prevent code duplication between timeseries & other modules (see the comments in lines 31-34 above). Do you think that's something that we should still do?

Reply:
Yes, lets sync on this later in the week during our 1:1 to see what else needs to be done. Ideally the bulk of trainer and learner logic should be moved to core. I'd probably focus on this in mid-late August / September.

Reply:
+1